src/main/java/hw3/components/DropDownMenu.java

```diff
@@ -0,0 +1,22 @@
+package hw3.components;
+
+import hw3.utils.Utils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+import java.util.List;
+
+public class DropDownMenu {
```

Comment:
Same story as for buttons and checkboxes elements.

Reply:
Renamed class